api.md

```diff
@@ -287,6 +287,15 @@ in the table.
 to how you handle constraints. A hypertable can contain foreign keys to normal SQL table
 columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must include the
 partitioning key.
+
+>:TIP: Running `create_hypertable` with the data migration on a table with foreign key contraints 
+might result in a deadlock. This is likely to happen when concurrent transactions simultaneously try 
+to insert data into tables, which are referenced in the foreign key constraints, and into the
+converting table itself. 
+The deadlock might abort `create_hypertable`. 
+>
+>The deadlock can be prevented by manually obtaining `SHARE ROW EXCLUSIVE` lock 
```

Comment:
I'm not sure this will be shown in the same TIP?  Also, do you want to give the syntax on how to do so, or pointer to postgres docs?

Reply:
cc: @solugebefola re: multi-paragraph :TIP: notation.

Reply:
> multi-paragraph :TIP: notation.

There is an example of Warning with multi-paragraph. I copied the usage from there

Reply:
> Also, do you want to give the syntax on how to do so, or pointer to postgres docs?

Fixed.